Web_Development/Javascript/Topics/Difference_Between_Const_Var_And_Let/README.md

```diff
@@ -0,0 +1,67 @@
+# **Difference Between Const , Var and Let**
+---
+
+## *Introduction:*
+Basically there are three types of variable declaration  Const, var and let in Javascript. We will learn some most common and important uses and differences among the three.
+
+---
+  1. **Const :** 
+  - Variables declared with the const maintain constant values.
+    
+  -  Const declarations can only be accessed within the block they were declared.
+
+  -  Const are hoisted to the top but are not initialized.
+  
+  -  One of the advantages of const is that it cannot be updated or re-declared. So if we declare in Const we can never do this-
+
+     > const saluation = "Hi Gwoc";          
+    saluation = "Hello Javascript";// error: Assignment to constant variable.   
+    const greeting = "Hello Opensource"; // error: Identifier 'saluation' has already been declared
```

Comment:
Please use code-blocks instead of quote blocks.

Correct:
````md
```js
someFunction();
```
````

Incorrect:
````md
> someFunction();
````